src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_LIMIT = 100;
+
+    private final Random random = new Random();
+
```

Comment:
There is no need to add empty lines between variables
```suggestion
    private final Random random = new Random();
```